python/llm/example/CPU/HF-Transformers-AutoModels/Model/README.md

```diff
@@ -24,6 +24,7 @@ You can use BigDL-LLM to run any Huggingface Transformer models with INT4 optimi
 | Aquila    | [link](aquila)    |
 | Replit    | [link](replit)    |
 | Mistral   | [link](mistral)   |
+| Flan-t5   | [link](flan-t5)   |
```

Comment:
Please also add links in https://github.com/intel-analytics/BigDL/blob/main/python/llm/README.md